Exception loading helper: /packages/velocity-jasmine-unit/lib/loader-helper.js

Hi I just played around with velocity and velocity-jasmine-unit.

I got mocha-web to work, but when I have test-jasmine-unit.js in the test folder I get the following error in the server console:

I20140513-23:50:16.615(2)? mocha-web: First test executed
I20140513-23:52:38.745(2)? Exception loading helper: /Users/ap/Documents/workspaces/VilangoWork/vicoding-v3/packages/velocity-jasmine-unit/lib/loader-helper.js
I20140513-23:52:38.746(2)? [ReferenceError: _ is not defined]

Any ideas what I'm doing wrong here?

Cheers,
Gerwin

When I run `npm install` in `packages/velocity-jasmine-unit` of my project then I get the following output:

```
npm ERR! install Couldn't read dependencies
npm ERR! package.json ENOENT, open '/Users/ap/.meteorite/packages/velocity-jasmine-unit/xolvio/velocity-jasmine-unit/47173dc3e6bae823b3a20b93382fe68a3f1e4b11/package.json'
npm ERR! package.json This is most likely not a problem with npm itself.
npm ERR! package.json npm can't find a package.json file in your current directory.

npm ERR! System Darwin 13.1.0
npm ERR! command "node" "/usr/local/bin/npm" "install"
npm ERR! cwd /Users/ap/.meteorite/packages/velocity-jasmine-unit/xolvio/velocity-jasmine-unit/47173dc3e6bae823b3a20b93382fe68a3f1e4b11
npm ERR! node -v v0.10.28
npm ERR! npm -v 1.4.10
npm ERR! path /Users/ap/.meteorite/packages/velocity-jasmine-unit/xolvio/velocity-jasmine-unit/47173dc3e6bae823b3a20b93382fe68a3f1e4b11/package.json
npm ERR! code ENOPACKAGEJSON
npm ERR! errno 34
npm ERR! 
npm ERR! Additional logging details can be found in:
npm ERR!     /Users/ap/.meteorite/packages/velocity-jasmine-unit/xolvio/velocity-jasmine-unit/47173dc3e6bae823b3a20b93382fe68a3f1e4b11/npm-debug.log
npm ERR! not ok code 0
```

--- a/.meteor/packages
+++ b/.meteor/packages
@@ -10,4 +10,4 @@ iron-router
 coffeescript
 velocity
 mocha-web
-velocity-jasmine-unit
+jasmine-unit

--- a/README.md
+++ b/README.md
@@ -38,7 +38,7 @@ and these lines to `.meteor/packages`
 ```
 
 ```
-mrt add velocity-jasmine-unit
+mrt add jasmine-unit
 em g:scaffold todos
 
 DEBUG=1 mrt

--- a/packages/.gitignore
+++ b/packages/.gitignore
@@ -3,3 +3,5 @@
 /velocity
 /mocha-web
 /velocity-jasmine-unit
+/jasmine-unit
+/mirror

--- a/smart.json
+++ b/smart.json
@@ -6,8 +6,8 @@
     },
     "mocha-web": {
       "git": "https://github.com/mad-eye/meteor-mocha-web",
-      "branch": "mtr"
+      "branch": "velocity"
     },
-    "velocity-jasmine-unit": {}
+    "jasmine-unit": {}
   }
 }

--- a/smart.lock
+++ b/smart.lock
@@ -9,9 +9,9 @@
       },
       "mocha-web": {
         "git": "https://github.com/mad-eye/meteor-mocha-web",
-        "branch": "mtr"
+        "branch": "velocity"
       },
-      "velocity-jasmine-unit": {}
+      "jasmine-unit": {}
     },
     "packages": {
       "iron-router": {
@@ -22,22 +22,27 @@
       "velocity": {
         "git": "https://github.com/xolvio/velocity",
         "branch": "master",
-        "commit": "e039c12470ab01b271fc1218c40a7abb92231986"
+        "commit": "7b901697c588ab4df8d11124333efdb99acb7571"
       },
       "mocha-web": {
         "git": "https://github.com/mad-eye/meteor-mocha-web",
-        "branch": "mtr",
-        "commit": "44a47d87437558d623709ee99b21a82cb54dcd43"
+        "branch": "velocity",
+        "commit": "969c937d53ce34e563e527b3c048386a1b808512"
       },
-      "velocity-jasmine-unit": {
-        "git": "https://github.com/xolvio/velocity-jasmine-unit.git",
-        "tag": "v0.1.0",
-        "commit": "3e7da16742f31250c67fb8b2f581b93824c6e9d6"
+      "jasmine-unit": {
+        "git": "https://github.com/xolvio/jasmine-unit.git",
+        "tag": "v0.1.2",
+        "commit": "bea45a66302e08043c6fad0a24011bfe047bc789"
       },
       "blaze-layout": {
         "git": "https://github.com/EventedMind/blaze-layout.git",
         "tag": "v0.2.4",
         "commit": "b40e9b0612329288d75cf52ad14a7da64bb8618f"
+      },
+      "mirror": {
+        "git": "https://github.com/xdissent/meteor-mirror.git",
+        "tag": "v0.0.2",
+        "commit": "03d965dcab2f976a1fdc6ce513edcd4c17b75746"
       }
     }
   }

--- a/tests/.gitignore
+++ b/tests/.gitignore
@@ -1,0 +1,1 @@
+.reports

--- a/tests/underscore-stub.js
+++ b/tests/underscore-stub.js
@@ -1,0 +1,4 @@
+_ = {
+  extend: function () {},
+  each: function () {}
+}

